python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -165,7 +165,8 @@ def _setParent(self, parent):
     def get(self, to_match):
         def matching_filter(item):
             if '_list_get__match__' in dir(item):
-                return item._list_get__match__(to_match)
+                ## This drops through to a base class so for sanity sake strip it !
+                return item._list_get__match__(stripProxy(to_match))
```

Comment:
But it's needed now for some other code I'm wanting to get in, when can we expect #393?